stringassignment.js

```diff
@@ -0,0 +1,42 @@
+// 1. `console.log` the first character of a string.
+let str = 'Hey how\'s it going?';
+console.log(str[0]);
+// 2. `console.log` the length of a string.
+let str2 = 'Not bad, how about you?';
+console.log(str2.length);
+// 3. `console.log` the last character of any string.
+let str3 = 'I\'m really looking forward for this summer';
+console.log(str3[str3.length - 1]);
+// 4. Create a code block that takes a single string variable,
+//and returns a copy of the string with the last letter capitalized.
+let str4 = 'giselle'
+str4 = str4.slice(0,6) + str4[6].toUpperCase()
```

Comment:
This is would be what we call hard coding. Your code must be able to accommodate for differing input. For this instance what would happen if we change str4 to "JRJ"? Always test your code with different input before moving on to next questions.